content/200-concepts/100-components/02-prisma-client/037-relation-queries.mdx

```diff
@@ -364,75 +320,40 @@ const getUser = await prisma.user.findUnique({
 })
 ```
 
-### Relation count
+## Relation count
 
 In [3.0.1](https://github.com/prisma/prisma/releases/3.0.1) and later, you can [`include` or `select` a count of relations](aggregation-grouping-summarizing#count-relations) alongside fields - for example, a user's post count.
 
-### Filter a list of relations
-
-When you use `select` to return a subset of data from related records (for example, a user's posts), you can **filter and sort that list of relations**. For example, you might want to query all users with a `prisma.io` email address and select the titles of their published posts.
-
-#### Return _all_ `User` records where `email` contains `prisma.io`, and select the `title` of all `Post` records that have not been published yet
+<CodeWithResult>
+<cmd>
 
-```js highlight=9-14;normal
-const result = await prisma.user.findMany({
-  where: {
-    email: {
-      contains: 'prisma.io',
-    },
-  },
-  select: {
-    posts: {
-      where: {
-        published: false,
-      },
-      orderBy: {
-        title: 'asc',
-      },
-      select: {
-        title: true,
-      },
+```ts
+const relationCount = await prisma.user.findMany({
+  include: {
+    _count: {
+      select: { posts: true },
     },
   },
 })
 ```
 
-You can use the `post` model property to write the same query:
```

Comment:
The content in this section is also contained in (a subsection close to the bottom of the page on Relation filters) https://github.com/prisma/docs/pull/4670/files#diff-4900bb40166500b306182e140816c922b9f2dab07a7be6102ee6d8a349994480R1024-R1026.

Reply:
I've restored the content in this section, and restored it right underneath the Relation count content